benchmarks/sha2.cc

```diff
@@ -90,30 +90,29 @@ HACL_Sha2_streaming(benchmark::State& state, Args&&... args)
 
   auto digest_len = std::get<0>(args_tuple);
   auto expected_digest = std::get<1>(args_tuple);
-  auto create_in = std::get<2>(args_tuple);
-  auto init = std::get<3>(args_tuple);
+  auto malloc = std::get<2>(args_tuple);
+  //auto reset = std::get<3>(args_tuple);
```

Comment:
Is this tested elsewhere or is this not needed anymore?

Reply:
reset is not required after a malloc (that is why the old name, init, was confusing).
but indeed, I don't think it is tested elsewhere...

Reply:
I filed #432 to track actually testing this.